hw/ip/otbn/doc/dv/index.md

```diff
@@ -325,6 +336,146 @@ No special coverage points for this instruction.
+
+#### BN.MULQACC
+
+- Cross `wrs1_qwsel` with `wrs2_qwsel` to make sure they are applied to the right inputs
+- See the accumulator overflow
+
+#### BN.MULQACC.WO
+
+As for `BN.MULQACC`, plus the generic flag group cover points.
+
+#### BN.MULQACC.SO
+
+As for `BN.MULQACC` plus the following:
+
+- See bits being cleared in the destination register as part of shifting out the result (makes sure that we're masking things properly when assembling the new value)
+- Cross the generic flag updates with `wrd_hwsel`, since the flag changes are different in the two modes.
+
+#### BN.SUB
+
+As for `BN.ADD`.
+
+#### BN.SUBB
+
+As for `BN.ADDC`.
+
+#### BN.SUBI
+
+As for `BN.SUB`.
+
+#### BN.SUBM
+
+- Execute with the two extreme values of `MOD` (zero and all ones)
+- A non-negative intermediate result with a nonzero `MOD` (so `MOD` is not added).
+- A negative intermediate result with a nonzero `MOD` (so `MOD` is added).
+- A very negative intermediate result with a nonzero `MOD` (so `MOD` is added, but the top bit is still set)
+- An intermediate result that exactly equals a nonzero `-MOD`.
+
+#### BN.AND
+
+- Extremal values of shift for both directions where the shifted value is nonzero
+- Toggle coverage of the output result (to ensure we're not just AND'ing things with zero)
+
+#### BN.OR
+
+- Extremal values of shift for both directions where the shifted value is nonzero
+- Toggle coverage of the output result (to ensure we're not just OR'ing things with zero)
+
+#### BN.NOT
+
+- Extremal values of shift for both directions where the shifted value is nonzero
+- Toggle coverage of the output result (to ensure nothing gets clamped)
+
+#### BN.XOR
+
+- Extremal values of shift for both directions where the shifted value is nonzero
+- Toggle coverage of the output result (to ensure we're not just XOR'ing things with zero)
+
+#### BN.RSHI
+
+No special coverage.
+
+#### BN.SEL
+
+- Cross flag group, flag and flag value (2 times 4 times 2 points)
+
+#### BN.CMP
+
+As for `BN.SUB`.
+
+#### BN.CMPB
+
+As for `BN.SUBB`.
+
+#### BN.LID
+
+- Load from a valid address, where `<grs1>` is above the top of memory and a negative `<offset>` brings the load address in range.
+- Load from a valid address, where `<grs1>` is negative and a positive `<offset>` brings the load address in range.
+- Load from address zero
+- Load from the top word of memory
+- Load from an invalid address (aligned but above the top of memory)
+- Load from a misaligned address
+- See an invalid instruction with both increments specified
+- See `<grd>` greater than 31, giving an illegal instruction error
+- Cross the three types of GPR for `grd` with `grd_inc`
+- Cross the three types of GPR for `grs1` with `grd_inc`
+
+#### BN.SID
+
+- Store to a valid address, where `<grs1>` is above the top of memory and a negative `<offset>` brings the load address in range.
+- Store to a valid address, where `<grs1>` is negative and a positive `<offset>` brings the load address in range.
```

Comment:
Nit: you're using `<reg>` here, and `reg` in other places.

Reply:
Wow, I've been really inconsistent! Dropping the angle brackets.